### ISSUE TITLE
[DropDownList] popup is reopened when the arrow icon is clicked in IE 11

### ISSUE THREAD

**Issue by mbechev** [telerik/kendo-angular#2652]
**Describe the bug**
When the arrow icon of the DropDownList is clicked, the popup is reopened in IE 11.

**To Reproduce**
1. Open in IE 11
1. Click the arrow icon.
1. Popup is reopened.
https://stackblitz.com/edit/angular-u9uyzy?file=app/app.component.ts

**Workaround**
Add the _unselectable=on_ attribute to the arrow icon:
``` ts
    public ngAfterViewInit(): void {
        document.querySelector('.k-select .k-icon').setAttribute('unselectable', 'on');
    }
````
https://stackblitz.com/edit/angular-u9uyzy-umwjww?file=app/app.component.ts

**Comment by siyka-marinova** [comment]
Fix for this issue is available in the [latest dev. version](https://www.telerik.com/kendo-angular-ui/components/installation/development-builds/#toc-obtaining-development-builds) of `kendo-angular-dropdowns`

**Comment by dimitar-pechev** [comment]
Released in **v4.2.6**.
